src/components/Config/Config.tsx

```diff
@@ -120,68 +146,148 @@ function Config() {
     }
   }, [checkConnection, wallet]);
 
+  const ArbitrumNetworkConnect = useCallback(async () => {
+    const ArbitrumTestChainId = '0x66eeb'; // arbitrum test net
+    if (typeof window.ethereum !== 'undefined') {
+      const chainId = await window.ethereum.request({ method: `eth_chainId` });
+      if (chainId === ArbitrumTestChainId) {
```

Comment:
This is not enough to know you're connected to Arbitrum via the proxy. You also need to check `isConnectedToAutomate` (see `useAutomateConnection`)